frontend/src/composables/errorService.ts

```diff
@@ -0,0 +1,36 @@
+import {reactive} from 'vue';
+import {AxiosError} from "axios";
+import {useToastService} from "@/composables/toastService";
+
```

Comment:
I know you started, before issue #6 was there - but that one defines a folder structure into features and shared with subfolders.
I suggest to move these compoables into shared/composables

Reply:
sure, will do